Dockerfile

```diff
@@ -24,5 +24,14 @@ WORKDIR /go/src/github.com/insolar/insolar
 RUN  set -eux; \
      groupadd -r insolar --gid=999; \
      useradd -r -g insolar --uid=999 --shell=/bin/bash insolar
-COPY --from=build /go/src/github.com/insolar/insolar/bin/insolar /go/src/github.com/insolar/insolar/bin/insolard /go/src/github.com/insolar/insolar/bin/keeperd /go/src/github.com/insolar/insolar/bin/pulsard /usr/local/bin/
+COPY --from=build /go/src/github.com/insolar/insolar/application/api/spec/api-exported.yaml /app/api-exported.yaml
 
+# add script and configs required for network bootstrap
+ADD scripts/kube/bootstrap/* /app/bootstrap/
```

Comment:
same

Reply:
Agreed, but we can left it here if it's not critical and create task to fix it, or if it's too nasty I can move this to `kube` repo. Your thoughts?

Reply:
sounds reasonable